ufs/ccpp/data/MED_typedefs.meta

```diff
@@ -1234,7 +1234,7 @@
   name = MED_typedefs
   type = module
   relative_path = ../../../../../FV3/ccpp/physics/physics
-  dependencies = machine.F,physcons.F90,physparam.f
```

Comment:
Why `physparam.f` is removed. Is it also removed in CCPP/Physics?

Reply:
Yes, Dustin asked me to remove this since it was causing some sort of failure (I don't remember the details).